modules/chapter1/pages/section6.adoc

```diff
@@ -0,0 +1,157 @@
+= KServe Custom Serving Example
```

Comment:
I think we could merge this section with section 3 to have a single custom serving section.